src/hugo/content/en-us/implementations/aws/deviceOnboarding/Provisioning-JITP/_index.md

```diff
@@ -0,0 +1,143 @@
+{{% notice note %}}
+To use JITP you have to maintain your PKI infrastructure, this can also be done using managed service like Amazon Certificate Manager. The manufacturing facility should have capability to create a private key, CA signed certificate and store the key-certificate pair on the device memory.
+{{% /notice %}}
+
+## Use cases
+Just-in-time provisioning (JITP) is the recommended approach when the manufacturing chain allows the device maker to provision unique credentials into the device at manufacturing time. To implement JITP the manufacturing chain has to be trusted and should have the capability to create a device certificate signed by a self-managed CA. You can have your devices provisioned when they first attempt to connect to AWS IoT with just-in-time provisioning (JITP). 
+
+Example use cases where JITP should be used are:
+
+1. When device originates from an original device manufacturer and you have the ability to include unique key material.
+2. When maintaining public key infrastructure (PKI) is feasible.
+3. There is a trust channel between your PKI and the manufacturing facility.
+
+## Reference architecture
+
+![JITP](JITP_arch_1.png)
+The details of this flow are as follows:
+1. Private key and signed certificate pair is created using PKI. PKI can be self-managed or using a managed service like Amazon ACM. 
+2. This pair is securely copied and stored on the device memory. 
+3. Using JITP, the device connects to AWS IoT, and the certificate’s signature is verified
+against the registered CA. After verification, a provisioning template registers the Thing,
+certificate, and assigns a policy to the device. A provisioning template is a JSON document that uses parameters to describe the resources your device must use to interact with AWS IoT. 
+4. When the device connects to AWS IoT Core for the first time, the device certificate, and
+the signer CA that is registered with AWS IoT must be sent during the TLS handshake.
+5. The TLS handshake will fail at the first connection. This happens because the certificate
+has not been pre-loaded into the AWS IoT account. 
+6. The device-supplied certificate is
+registered and activated in AWS IoT during the provisioning process. 
+7. The device must
+have logic to reconnect to AWS IoT after a short time period. 
+8. If the provisioning
+operation has succeeded, the device will connect to AWS IoT successfully.
+
+Below diagram describes end-to-end just-in-time provisioning flow:
+
+![JITP](JITP.png)
+
+
+## Implementation
+To use JITP you need to have control over your Certificate Authority. This can be either done by self-managed root CA or using ACM Private CA. ACM Private CA enables creation of private certificate authority (CA) hierarchies, including root and subordinate CAs, without the investment and maintenance costs of operating an on-premises CA. Your private CAs can issue end-entity X.509 certificates useful in scenarios including:
```

Comment:
Instead of self-managed root CA, maybe issuing CA? Some users may have a root CA while other use suboridinates.